spec/requests/providers/shared_ownerships_spec.rb

```diff
@@ -26,6 +26,22 @@
         expect(unescaped_response_body).to match('Does your client own their home with anyone else?')
         expect(unescaped_response_body).to match(LegalAidApplication::SHARED_OWNERSHIP_YES_REASONS.first)
       end
+
+      describe 'back link' do
+        context 'applicant owns with mortgage' do
+          it 'points to property_value page' do
+            expect(response.body).to have_back_link(providers_legal_aid_application_outstanding_mortgage_path(legal_aid_application))
```

Comment:
I think the description of this test is not correct.
Should it be it 'points to the outstanding_mortgage page' do?

Reply:
yep